### ISSUE TITLE
니모닉 인식 문제

### ISSUE THREAD

**Issue by darkcircle** [changwoo/ko-po-check#28]
https://l10n.gnome.org/vertimus/gnome-calculator/master/help/ko#306512

gnome-calculator.help.master.ko.po 에서 

```
#. (itstool) path: info/title
#: C/superscript.page:7
msgctxt "sort"
msgid "_"
msgstr "_"
```

부분을 니모닉으로 인식(false-positive + exception)하는 문제가 있습니다.

```
Traceback (most recent call last):
  File "/usr/bin/ko-po-check", line 119, in <module>
    errors = checker._check(entry, filtercontext[filtername])
  File "/usr/lib64/python3.3/site-packages/KPC/classes.py", line 34, in _check
    return self.check(entry, context)
  File "/usr/lib64/python3.3/site-packages/KPC/checks/terminology/desktop-commons.py", line 88, in check
    msgid_m = find_mnemonic(msgid)
  File "/usr/lib64/python3.3/site-packages/KPC/checks/terminology/desktop-commons.py", line 75, in find_mnemonic
    return msgid[i+1].upper()
IndexError: string index out of range
```

KPC/checks/terminology/desktop-commons.py 에서

```
def find_mnemonic(msgid):
    try:
        i = msgid.index('_')
        return msgid[i+1].upper()
    except ValueError:
        return None


class DesktopCommonsCheck(BaseCheck):
    errstr = '%s: 다음과 같이 번역해야 합니다: \"%s\"'

    def check(self, entry, context):
        msgid = entry.msgid
        msgstr = entry.msgstr
        errors = []
        msgid_n = normalize_msgid(msgid)
        msgid_m = find_mnemonic(msgid)
        msgid_d = (msgid[-3:] == '...')
        for (orig, trans) in data:
            # 모두 대문자로 된 단어는 option argument 따위로 쓰이므로 넘어간다
            if msgid_n.lower() == orig and not msgid_n.isupper():
                good_msgstr = make_msgstr(trans, msgid_m, msgid_d)
                if good_msgstr != msgstr:
                    errors.append(Error(self.errstr % (msgstr, good_msgstr)))
        return errors
```

find_mnemonic() 을 실행할 때 i = msgid.index('_') 실행 부분에서 나온 인덱스 값이 msgid.length - 1 일 경우 None을 반환하면 이 문제를 해결할 수 있을 것 같습니다.

**Comment by changwoo** [comment]
47f7e0d067192973c6976b228cb3c4b8144b42a1 커밋에서 exception 처리.
